conf.d/yarn.yaml.example

```diff
@@ -17,7 +17,12 @@ instances:
     # A Required friendly name for the cluster.
     # cluster_name: MyYarnCluster
 
+    # Tags from running applications that will be applied for application metrics, set as app_key:datadog_tag
```

Comment:
After seeing the code I understand what this means, but it could probably be clearer by adding an example (the `name:app_name` one), and a link indicating what can be used to tag applications.